discoveries/houston-aod.discoveries.mdx

```diff
@@ -0,0 +1,84 @@
+---
+id: 'houston-aod'
+name: Aerosol Optical Depth
+description: "The Impact of Aerosols on the Houston, TX Metro"
+media:
+  src: ::file ./steam-power-plant.jpg
+  alt: Smokestack from Power Plants
+  author:
+    name: Maxim Tolchinskiy
+    url: https://unsplash.com/photos/W3y2crFkVIs
+pubDate: 2023-05-23
+thematics:
+  - air-quality
+---
+
+<Block>
+  <Prose>
+    ### Socioeconomic Vulnerability
+
+    Figure 1 illustrates the decadally-averaged AOD from 2000-2009 and 2010-2019 with socioeconomic vulnerability. Individuals of a less privileged socioeconomic status are at higher risk for complications from air pollution than those that of a higher status. However, one of the primary reasons why those a lower status are more susceptible to complications are due to these residencies being located in and around industrial areas.
+  </Prose>
+  <Figure>
+    <Image
+      src={new URL('./houston-aod-fig.png', import.meta.url).href}
+      alt='AOD socioeconomic (Houston)'
+    />
+    <Caption>
+      Figure 1: Socioeconomic status plots showing the change in AOD values over the last 20 years.
+    </Caption> 
+  </Figure>
+</Block>
+
+<Block>
+  <Prose>
+    ### Implications for Aerosol Optical Depth
+
+   Aerosols can have a variety of implications for the general public. In terms of air quality, aerosols can affect the air quality from high concentrations of pollutants that may pose a risk to public health. The highest risk comes from the power plants where the level of emissions are the highest. However, in recent years, pollutions sources have been regulated by the Environmental Protection Agency (EPA) and has found ways to reduce the amount of emissions.
+
+   Using satellite data from the MODIS Terra and Aqua instruments that observe Earth at different times of the day and provide a global view 1-2 days. This satellite provides crucial information such as Aerosol thickness that aides in determing the trend in aerosol patterns to help determine the track for future pollution. 
+  </Prose>
+</Block>
+
+<Block type="full">
+  <Figure>
+    <Map
+      datasetId='aerosol-dataset'
+      layerId='houston-landcover'
```

Comment:
are you trying to use the `houston-landcover` or `houston-aod` layer? the `houston-landcover` layer doesn't exist in the `aerosol-dataset` dataset. It's probably in some other dataset